xgi/classes/reportviews.py

```diff
@@ -84,8 +84,15 @@ def __init__(self, id_dict, id_attr, ids=None):
 
     @property
     def ids(self):
-        """The ids in this view."""
-        return list(self._id_dict.keys()) if self._ids is None else self._ids
+        """The ids in this view.
+
+        Notes
+        -----
+        DO NOT use this property for membership check: instead of `x in view.ids` always
```

Comment:
why?

Reply:
Because this is slower. I updated the docs to reflect this fact.